docker-compose.functional-tests.yml

```diff
@@ -6,20 +6,21 @@ services:
       command: test:integration --grep @citizen
       environment:
         - WEB_DRIVER_HOST=remote-webdriver
-        - CITIZEN_APP_URL=${TEST_URL}
```

Comment:
TEST_URL is used locally for nightly build and security, please search and change in all occurrences .

Reply:
I will look at that in a seperate PR. However, TEST_URL still available in the Jenkins Library to be used.